src/canisters/frontend/ic-certified-assets/src/lib.rs

```diff
@@ -1,9 +1,12 @@
 //! This module declares canister methods expected by the assets canister client.
 pub mod asset_certification;
+mod canister_env;
+mod cookies;
 pub mod evidence;
 pub mod state_machine;
 pub mod types;
 mod url_decode;
+mod url_encode;
```

Comment:
```suggestion
mod url_encode;
```
Nit: Those are two extremely small modules, maybe we can collate them so that it's also easier to see they use en/decode logic that is compatible between the two functions?